web/website/content/posts/2023-05-07-whats-next-for-prql.md

```diff
@@ -0,0 +1,118 @@
+- Though the Developers aspect of the three-legged stool has consistently been
+  the strongest, even here we've seen a decline in engagement:
+  - The past few months have had fluctuating levels of contributions outside the
+    core team — with a surge in Q1, but slowing since then.
+  - Recently, even the core team's pace has decreased (although there's always
+    variance, and I'm not suggesting a quick fix by us working harder).
+
+## Finding the Flywheel
+
+The most crucial step is enabling people to use PRQL. Much will follow — we'll
+know which features and bugs to tackle, be able to showcase satisfied users, and
+so on.
+
+Some options:
+
+- Revisit `dbt`. I had envisioned this as a particularly fruitful integration —
+  for both PRQL and dbt.
+
+  - The monkeypatch plugin couldn't technically achieve everything needed.
+  - I also failed to have the dbt-core PR merged — it received a brief review
+    before being dropped.
+  - The dbt team remains interested in merging an improved version of the
+    dbt-core plugin (but won't guarantee a merge, which I can understand, though
+    makes it a less compelling investment)
+  - I'm hesitant to invest more time given this track record but open to it if
+    others believe it's a worthwhile bet.
+
+- Enhance language-specific integrations
+
+  - The Jupyter integration is "OK". Python is very popular so if it were
+    possible to make it excellent, then it could be great. But can it be great?
+    Can we get good editor features in a notebook? I _think_ we can get syntax
+    highlighting.
+    - TODO: how much integration can we actually do in Jupyter? Could we have an
+      LSP?
+  - The R extension looks great. I don't have much experience with R and haven't
+    explored it extensively. Do others have opinions? Is the R audience enough
+    to make a project-wide bet on R?
+
+- Create our own tool. This gives us complete control.
+
+  - But unless it's something exclusive to PRQL, are we likely to build
+    something superior to what's available?
+  - We're already asking people to bet on PRQL — is asking them to gamble on
+    both a new language and a new tool likely to work?
+  - Generally, successful things are smaller than we anticipate, not larger.
+  - Is there something PRQL can do that SQL can't, which would provide more
+    justification for building our own tool? Perhaps something related to
+    pipelines and instant feedback (e.g. the playground displaying data results
+    is quite impressive and could be expanded)?
+
+- Fork Rill
+
+  - Rill seems to be an especially good match for PRQL — enabling quick
+    exploration and immediate results.
+  - However, I didn't receive a response from the team in my latest email
+    (though I can follow up again). We had a productive Zoom meeting previously.
+  - We could consider forking the project if necessary, and then using any
+    success there to attempt to merge it to mainline.
```

Comment:
As I noted, forking Rill would be a temporary boost to the project, but would not showcase strengths of PRQL as it would be hard to integrate good highlighting and LSP support.